ubuntu/ubuntu-24.04.pkr.hcl

```diff
@@ -81,7 +81,7 @@ build {
       "org.openstack.glance.os_version" = "24.04"
       "org.openstack.glance.os_type" = "linux"
       "org.openstack.glance.architecture" = "x86_64"
-      "org.opencontainers.image.title" = "EGI Ubuntu 24.04 image"
+      "eu.egi.cloud.description" = "EGI Ubuntu 24.04 image"
```

Comment:
This will trigger a rebuild of the image, so I would change the version. Or maybe we should force the version to use the commit sha so it's clear the change?
Then we would have:
`ubuntu:24.04`
`ubuntu:24.04-$(commit sha)`
`ubuntu:24.04-latest`
`ubuntu:latest` <- this one needs to be flagged with this custom data

Reply:
Even if manual, IMO the current approach is more intuitive:
`ubuntu:24.04` <- always pointing to the latest
`ubuntu:24.04-<date>`
This way it is easier to check how old is the VMI you are using.
But that's just my opinion :)
If you agree, I would just change the version, which I forgot. Thanks!